From 5f10bd9b64c745084292a7094af2e4c3d14c1ac5 Mon Sep 17 00:00:00 2001
From: Arunachalam Ganapathy <arunachalam.ganapathy@arm.com>
Date: Tue, 12 Apr 2022 12:43:49 +0100
Subject: [PATCH 1/3] arm: total_compute: update secure dram size

Update secure DRAM size as it is increased by 64MB for additional
secure partitions.

Signed-off-by: Arunachalam Ganapathy <arunachalam.ganapathy@arm.com>
Change-Id: Id8ce99c7a5330d3c28d473009c4db04141e6fa4d
Upstream-Status: Pending [Not submitted to upstream yet]
---
 include/configs/total_compute.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/total_compute.h b/include/configs/total_compute.h
index 933a145f99..66de8ae239 100644
--- a/include/configs/total_compute.h
+++ b/include/configs/total_compute.h
@@ -25,8 +25,8 @@
 
 /* Physical Memory Map */
 #define PHYS_SDRAM_1		0x80000000
-/* Top 48MB reserved for secure world use */
-#define DRAM_SEC_SIZE		0x03000000
+/* Top 112MB reserved for secure world use */
+#define DRAM_SEC_SIZE		0x07000000
 #define PHYS_SDRAM_1_SIZE	0x80000000 - DRAM_SEC_SIZE
 #define CONFIG_SYS_SDRAM_BASE	PHYS_SDRAM_1
 
-- 
2.30.2

